python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -324,16 +326,17 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
                 if len(forced) != len(refCat):
                     raise RuntimeError(("Lengths of forced (N = {0:d}) and ref (N = {0:d}) cats don't match").
                                        format(len(forced), len(refCat)))
-                refColList = [s for s in refCat.schema.getNames() if
+                refCatSchema = getSchema(refCat)
+                refColList = [s for s in refCatSchema.getNames() if
```

Comment:
I thought from the commit description that `getSchema` returned a list of columns; if so, then `refCatSchema.getNames()` doesn't make sense.  Maybe it will make sense later on.